Jenkinsfile

```diff
@@ -14,7 +14,7 @@
 pipeline {
 
     agent {
-        label joyCommonLabels(image_ver: '18.4.0')
+        label joyCommonLabels(image_ver: '21.4.0', pi:'20210826T002459Z')
```

Comment:
Our 21.4.0 agents have been hand-hacked to fix the joyent things that are actually in the 21.4.1 pkgsrc image. I'll mention 21.4.0 vs 21.4.1 again below.